### ISSUE TITLE
Loading ImmGen_T.Nve.Sp.ATAC.bg plus hic_bedpe.txt leads to an infinite loop of connecting and disconnecting

### ISSUE THREAD

**Issue by alosdiallo** [alosdiallo/HiC_Network_Viz_tool#48]
If you load ImmGen_T.Nve.Sp.ATAC.bg  then later try to load another dataset via upload the app goes into an infinite loop. Where it tries to connect then disconnect over and over again.

**Comment by alosdiallo** [comment]
This happens when I try to submit perimeters as well.

**Comment by KHBHH** [comment]
What I understood from trying this dataset, that this fails because it is not reading ATACdata properly from the beginning. I know it sounds strange, but when you press process button in this case, the loading is happening very quickly, and it should not be that quick, It should take time. When I tried to press process button I was able to see hic data in table but not ATAC, this means i t did not read preporly, hence when you try to plot it, it will fail. 

I tried to run this on R to have a track of what is happening, and I got the following **Warning: Error in eval: object 'ATACdata' not found**. 

On the other hand when I try ATAC(Immgen) and Chip-seq it works fine.

**Comment by rnr-HMS** [comment]
Yes, i was also getting this issue when loading the ATAC second. If we cannot readily resolve this before submission. We can include a disclaimer where we tell users to load .bg files first if visualizing them with other data types.

**Comment by KHBHH** [comment]
I took a small part of Immgen data that includes chrom1 only. It reads fine, there is no issue which data type is first read. I think it has to do something with the size of the file again.
